script/migrate-test-e2e.js

```diff
@@ -1,7 +1,14 @@
 import chalk from "chalk";
 import { $, execaCommand } from "execa";
+import { createVitest } from "vitest/node";
+
+const vitest = await createVitest("test", {
```

Comment:
Ping @Aslemammad - the `await vitest.start(); await vitest.exit();` seems to work. Nice!

But going a step further, is there anything stopping this file from being a Vitest test itself? Like, being able to run `vitest script/migrate-test/e2e.js` and use `describe`/`expect`/etc. inside this file itself? That way we wouldn't have to manually use the Vitest Node API.

Reply:
My only concern is that running vitest on the whole project would also include this file part of the tests! unless we exclude it and use another vitest command for it! Since this file has so many mutations, this makes me even more concerned. 
As far as I know, the tests in the scripts/ are different from the general tests.

Let me know your opinion if I should keep it or try the new command way.

Reply:
> exclude it and use another vitest command for it!

Yeah you can customize the `-t`/`--testNamePattern` Vitest runs with: https://vitest.dev/guide/cli.html#options. The `test:migrate` command could specify one that only runs this file, not the others.